cluster-state-service/handler/reconcile/loader/translate.go

```diff
@@ -54,6 +53,10 @@ func ToTask(ecsTask ecs.Task) types.Task {
 		stoppedAt := ecsTask.StoppedAt.Format(timeLayout)
 		taskDetail.StoppedAt = stoppedAt
 	}
+	if ecsTask.Version == nil {
```

Comment:
I can't think of a scenario where this would ever happen. It was just handling an "in case this happens" scenario. I'll remove it, as I agree that it is probably an unnecessary check.